gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -416,6 +416,27 @@ gst_tensor_filter_parse_modelpaths_string (GstTensorFilterProperties * prop,
   }
 }
 
+/**
+ * @brief check if the allocate_in_invoke is valid for the framework
+ * @param[in] priv Struct containing the properties of the object
+ * @return TRUE if valid, FALSE on error
+ */
+gboolean
+gst_tensor_filter_allocate_in_invoke (GstTensorFilterPrivate * priv)
+{
+  int allocate_in_invoke;
+
+  allocate_in_invoke = priv->fw->allocate_in_invoke;
```

Comment:
Hmm, I think `allocate_in_invoke` is now use-less because `allocateInInvoke` may decide this (i.e., NULL or not)

Reply:
Nah.. this is for the backward compatibility. With V1 API, both becomes useless :)

Reply:
Oh got it